### ISSUE TITLE
Maintain

### ISSUE THREAD

**Issue by NMW03** [twitterjs/twitter.js#4]
Are you going to maintain this. Because I'm working on my twitter library and saw you got `twitter.js` on npm

**Comment by iShibi** [comment]
Yes, I've been working a lot on this locally and will definitely maintain it. Anyway, best wishes for your library.
